vscode-extension/HACKING.md

```diff
@@ -0,0 +1,15 @@
+## Developing
+
+Run `npm run install-extension` to build and install the extension in your local VSCode. If you run `npm run install-extension --watch` it will rebuild and reinstall on every change, though you'll need to run `Developer: Reload Window` in VSCode to pick up on the changes.
```

Comment:
I believe this script is only defined in the vscode-extension subdirectory, so probably should mention that